setup.py

```diff
@@ -30,6 +32,7 @@
         'ckanapi.tests.mock',
         'ckanapi.cli',
         ],
+    setup_requires='requests',
```

Comment:
ok, please add a comment to this effect on near this line, because without knowing that it doesn't make much sense!